spec/expression.dd

```diff
@@ -91,12 +91,16 @@ $(H3 $(LNAME2 assignment_operator_expressions, Assignment Operator Expressions))
         a = cast(typeof(a))(a op b)
         --------------
 
-    except that:
+    except that
 
     $(UL
-        $(LI operand $(D a) is only evaluated once)
-        $(LI overloading $(I op) uses a different function than overloading $(I op)= does)
-        $(LI the left operand of $(D >>>=) does not undergo integral promotions before shifting)
+        $(LI operand $(D a) is only evaluated once,)
+        $(LI overloading $(I op) uses a different function than overloading $(I op)= does, and)
+        $(LI the left operand of $(D >>>=) does not undergo integral promotions before shifting.)
+    )
+    
+    $(P For user-defined types, assignment operator expressions can be overloaded separately from
+        the binary operator. Still the left operand must be an lvalue.
```

Comment:
Right. I wonder why there is no mixin template in Phobos for doing these correctly and en masse.
With *can* I meant there could be an overload that does not behave as the example does.
I'll fix the wording.